src/unstract/sdk/audit.py

```diff
@@ -68,13 +68,16 @@ def push_usage_data(
         execution_id = kwargs.get("execution_id", "")
         adapter_instance_id = kwargs.get("adapter_instance_id", "")
         run_id = kwargs.get("run_id", "")
-
+        llm_usage_reason = ""
+        if event_type == "llm":
+            llm_usage_reason = kwargs.get("llm_usage_reason", "")
```

Comment:
@Deepak-Kesavan tied to [this comment](https://github.com/Zipstack/unstract/pull/413#discussion_r1648030720), can it be `usage_reason` in an attempt to be generic?